src/System.Data.Extensions/ref/System.Data.Extensions.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
```

Comment:
Can you please generate this with our tooling? You can do that by running the following target from your src project `msbuild /t:GenerateReferenceSource`.